snapcraft.yaml

```diff
@@ -25,6 +25,9 @@ description: |
 grade: stable
 confinement: strict # use 'strict' once you have the right plugs and slots
 
+environment:
+  LD_LIBRARY_PATH: $SNAP/usr/lib64:$SNAP/usr/lib:$SNAP/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}
```

Comment:
Why is that needed now? Would be nice to add a comment explaining why that's not handled by the desktop launcher

Reply:
It was needed because, by default, autotools plugin install libraries at /usr/lib64 instead of /usr/lib/x86_64-linux-gnu. I fixed that, so it's not needed anymore.